currentScript.js

```diff
@@ -2,27 +2,55 @@
 
 // MIT license
 
-(function(document){
-  var currentScript = "currentScript",
-      scripts = document.getElementsByTagName('script'); // Live NodeList collection
+(function (document) {
+  var findInlineScript = function findInlineScript(scripts, line) {
```

Comment:
Very clever! Can we improve perf here a bit by memoizing per script element and only re-running the regexp if the `outerHTML` string changes?

Reply:
Moved `outerHTML` and regexp outside of the loop so that it only ever gets processed once per `currentScript` call. Also now populating the variables only if the script turns out to be inline.